### PR TITLE
Work order style updates

### PR DESCRIPTION
**Pull request by nathf**

### CODE REVIEW

File: app/templates/macros/work_order_link.html [hunk 1]
```diff
@@ -34,20 +34,18 @@
...
       </td>
-      <td>
-        <span>
-          {% if value is not string %}
-            <a href="{{ url }}">{{ question.get("text") }}</a>
-          {% else %}
-            {{ value }}
-          {% endif %}
-        </span>
+      <td{% if value is string %} class="summary-item-free-text"{% endif %}>
```
Comment (matt-sm) [inline app/templates/macros/work_order_link.html]: could just put the <td> in the if and just have one if/else

File: app/buyers/views/work_orders.py [hunk 2]
```diff
@@ -74,12 +74,12 @@ def create_new_work_order(framework_slug, lot_slug, brief_id):
         )['workOrder']
 
     except APIError as e:
+        form.errors['seller'] = [e.message]
```
Comment (matt-sm) [inline app/buyers/views/work_orders.py]: does that need to be a list [e.message] - could it just be e.message ?

Comment (nathf) [inline app/buyers/views/work_orders.py]: The way `_form_errors.html` is written, if I pass it just as a string each letter will be displayed as its own `<li>`. Wrapping in a list kept it to one line item

File: app/templates/macros/work_order_link.html (hunk 1 above)
Comment (nathf) [inline app/templates/macros/work_order_link.html]: Updated

**Comment by matt-sm** [comment]
:+1:

**Pull request merged**